### PR TITLE
add io.github.jazztickets.openflap

### PR DESCRIPTION
**Pull request by jazztickets**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by jazztickets** [comment]
bot, io.github.jazztickets.openflap

**Comment by jazztickets** [comment]
bot, build io.github.jazztickets.openflap

**Comment by jazztickets** [comment]
bot, build io.github.jazztickets.openflap

**Comment by jazztickets** [comment]
ARM is untested and unsupported. You have been warned.

File: io.github.jazztickets.openflap.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: openflap
+rename-desktop-file: openflap.desktop
+rename-icon: openflap
+rename-appdata-file: openflap.xml
+
+finish-args:
+  - --device=all
```
Comment (nedrichards) [inline io.github.jazztickets.openflap.yml:11]: I assume for access to joysticks, controllers etc. correct?

Comment (jazztickets) [reply io.github.jazztickets.openflap.yml:11]: correct

**Pull request closed without merging**